Removed 'Azure' as deploy provider mention'

pantheredeye opened the pull request:
Azure was mentioned as a deploy provider in Chapter 0, but we don't have a deploy command for it. This removes that mention.

Josh-Walker-GM commented:
I'm not sure what correspondence this list was supposed to have with our specific deploy commands. For example we don't have a specific command to deploy to google cloud or aws those are instead available by using deploy providers that we do provide specific commands for.

Maybe @thedavidprice, @cannikin or @ahaywood would be best placed to confirm what impression we are trying to give with this list?

pantheredeye commented:
Ah. Yeah didn't think about checking others!! Hah. I focused on azure after searching for a user question about it. 

Screenshot of the full section for context: 

![image](https://github.com/user-attachments/assets/e8da3333-f45b-478f-9368-7a927334e8f6)

ahaywood commented:
I wonder if it would make more sense to update that list to include links out to the service (what it does now) _and_ a link to instructions for each provider. 

(For my own reference: here's [a link to the page screenshotted](https://docs.redwoodjs.com/docs/tutorial/chapter0/what-is-redwood/#deployment) above)

cannikin commented:
Hmmm...going through a blame, that whole second was written by me in one go, but I would have thought I'd only add providers we actually have `yarn rw deploy` commands for. The commit says "co-authored by David Price"...maybe it just appends that if he merged it? 

But, I could see him wanting to add as many deploy hosts as we theoretically *could* deploy to...I believe Serverless.com would let you deploy to those, so I could see him wanting to see them listed explicitly (since serverless.com isn't itself a host, just tooling to get to multiple hosts).

pantheredeye commented:
We can blend both approaches and move this one on. We can update the language to something like 
"Redwood has generators that make deployment a breeze, such as to the following platforms.... " 

We could then reference baremetal as we already do. Finally, we could make a blurb about how "you are free take deployment into your own hands like any other react app and deploy on your own terms to any provider such as ... "

What about including docker listing in our deploy section or with the baremetal mention?

cannikin commented:
> What about including docker listing in our deploy section or with the baremetal mention?

The docker option works for sure? Like to any server, or just fly.io maybe?

pantheredeye commented on docs/docs/tutorial/chapter0/what-is-redwood.md:
```diff
@@ -64,6 +64,10 @@ You can also prerender pages that contain variables pulled from the URL, like th
+
+#### Accessibility
+
+Redwood includes a couple of components to [aid screen readers](https://redwoodjs.com/docs/accessibility) in properly navigating your app. The `<RouteAnnouncement>` component tells a screen reader to read something aloud, even though it isn't visible in the browser. And the `<RouteFocus>` tells a reader to skip verbose navigation options at the top of a page and get to the content.
```
Moved "**Accessibility**" into the "**_Router_**" section, from below, because the transition sentence from the previous paragraph does not make sense leading into "_**Accessibility**_." See red line. Moving it up to the router section made the most sense out of the options I saw.
![image](https://github.com/user-attachments/assets/012c85d9-bd3b-43ae-a606-0a5517992755)

pantheredeye commented:
@cannikin requesting review - this should be good to go. I simply copied the deploy providers list from our deployment intro! I added the baremetal deploy note as a bullet to keep it in the providers group. Seemed to make sense to me. Also, I moved the "Accessibility" section up a few heading (see comment above) and made some grammatical-ish changes.

pantheredeye commented:
ooh this doesn't make the same changes to the other version docs! :/ I can get to that at some point this week.

pantheredeye commented:
@cannikin ok this PR should be good for a final review and merge. 

* The provider list change is ported back to v5 docs. 
* I removed 'docker' mention from v5-7 because it was merged in v8 as a feature. 
* I did not make the small typo corrections in the older docs, only the current branch of docs. They were small enough and I assume they will be in main moving forward.

cannikin commented:
@pantheredeye Getting an error from Prettier during the CI stuff...can you see if you can fix?

<img width="650" alt="image" src="https://github.com/user-attachments/assets/a8a58f4d-08c0-429e-9745-0edb317c39f3">

pantheredeye commented:
@cannikin I think I have it corrected.

cannikin commented:
Hmm, isn't that uploads doc live on the real site? Is it okay to delete in this PR?

pantheredeye commented:
The uploads plural is the correct doc. I think I made that one the day I was reviewing the doc for Danny. Sort of dirtied up my local codebase with it. I noticed it in another branch this weekend and then caught it in this one.

pantheredeye commented:
Let me know what you think. I can bike this and start over with a clean pr if you prefer.

cannikin commented:
Looks good to me!
